src/dps-validation-service/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/dfcms/ValidationController.java

```diff
@@ -43,7 +43,7 @@ public ValidationController(CaseService caseService) {
      *  @param driversLicense
      * @param surcode
      */
-    @RequestMapping(value = "/GetValidOpenDFCMSCase", produces = MediaType.APPLICATION_XML_VALUE, method = RequestMethod.GET)
+    @GetMapping(value = "/GetValidOpenDFCMSCase", produces = MediaType.APPLICATION_XML_VALUE)
```

Comment:
is this changing functionality?

Reply:
It is essentially the same it is just the recommended way. Different sonar scans will actually pick it up.